src/GlobalStyles.tsx

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+
+import { Colors } from './Colors';
+
+const baseStylesheet = StyleSheet.create({  
```

Comment:
What's the purpose of separating this from the export below? I've never done much CSS management so just a curiosity question and not a critique.

Reply:
The purpose was so that I could use the definition within other defined styles in the stylesheet (e.g. label started with the text definition and added to it).  A mixin type pattern.  And then I included in the exported  stylesheet as well so it could still be used elsewhere.  But- I did delete this file in lastest commit while implementing the new stylesheet pattern.

Reply:
Ack. :)